Class-Code/OOP/object-classes/jkohl/sketch.js

```diff
@@ -5,35 +5,58 @@
 /* A brief description of what the program is */
 
 
-let x,y,r;
-let xVel = 5;
-let yVel = 5;
+class Bubble {
+    constructor(_x, _y, _r) {
+        this.x = _x;
+        this.y = _y;
+        this.r = _r;
+    }
+    display(){
+        fill(200);
+        ellipse(this.x, this.y, this.r)
+    }
+    move(){
+        this.x += random(-width / 100, width / 100);
```

Comment:
This is code from Ben. Just be clear about where you are getting your source code from